packages/shared/lib/core/profile-manager/api/recoverAccounts.ts

```diff
@@ -1,8 +1,16 @@
 import { get } from 'svelte/store'
-import { profileManager } from '../stores'
+
+import { AccountSyncOptions } from '@iota/wallet'
+
 import { IAccount } from '@core/account'
 
-export async function recoverAccounts(accountGapLimit: number, addressGapLimit: number): Promise<IAccount[]> {
+import { profileManager } from '../stores'
+
+export async function recoverAccounts(
+    accountGapLimit: number,
+    addressGapLimit: number,
+    syncOptions?: AccountSyncOptions
```

Comment:
Because all of the parameters are optional in the `AccountSyncOptions` I decided to make it optional here for when we use it app-side.

Reply:
Yeah, makes a lot of sense!